include/seqan/arg_parse/arg_parse_argument.h

```diff
@@ -136,7 +136,6 @@ class ArgParseArgument
         OUTPUT_FILE,  // .. an outputfile (implicitly also a string)
         INPUT_PREFIX, // .. an inputprefix (implicitly also a string)
         ///@deprecated use INPUT_PREFIX instead
-        [[deprecated("Use INPUT_PREFIX instead.")]]
```

Comment:
Hm, it works by just placing the compiler hint after the value:
```c++
#include <iostream>

enum FOO
{
    BAR,
    BAX [[deprecated]],
    BAT
};

int main()
{
    FOO foo{BAX};
    return 0;
}
```
But of course this will probably not be detected if people are using static_casts or something like that.